### PR TITLE
#38688: Handles 10.0v2+ releases of Nuke.

### PR DESCRIPTION
**Pull request by thebeeland**
Nuke no longer makes use of the Quicktime desktop application when encoding .mov files as of the 10.0v2 release. We need to account for the resulting changes in the Write node as a result.

### CODE REVIEW

File: hooks/codec_settings.py [hunk 1]
```diff
@@ -53,3 +42,18 @@ def get_quicktime_settings(self, write_node, **kwargs):
                 # the 'codec' knob name was changed to 'format' in Nuke 7
                 write_node["file_type"].setValue("ffmpeg")
                 write_node["format"].setValue("MOV format (mov)")
+        elif nuke.NUKE_VERSION_MAJOR >= 10 and nuke.NUKE_VERSION_RELEASE > 1:
```
Comment (josh-t) [inline hooks/codec_settings.py:45]: Since the conditional is specific here, might be worth leaving a comment that explains why the change in behavior. Probably just cut/paste from the ticket or PR.

Comment (thebeeland) [reply hooks/codec_settings.py:45]: Commented!
